scripts/compare_warehouse_tables.py

```diff
@@ -0,0 +1,244 @@
+    has_timestamp = False
+
+    # Initializing the more_info variable that will output extra information of the tables
+    more_info = ""
+
+    # Looping through all the table names for columns and rows match, and extra rows in Dev and prod
+    for table_name in table_names:
+        row_result = []
+        time_iteration = True
+        not_matched = False
+        not_matched_info = ""
+        newrows = [table_name]
+        while time_iteration:
+            # Checking if table has a timestamps and (if applicable) looping through today, yesterday, one month ago, and six month ago
+            if table_name in tables_with_timestamps:
+                if time_count == 0:
+                    has_timestamp = True
+                    not_matched = False
+                    not_matched_info += ""
+                    timestamp_DevData = f" WHERE Date(allDevData.{tables_with_timestamps[table_name]}) = '{today.date()}'"
+                    timestamp_ProdData = f" WHERE Date(allProdData.{tables_with_timestamps[table_name]}) = '{today.date()}'"
+                if time_count == 1:
+                    one_day = today - timedelta(days=1)
+                    timestamp_DevData = f" WHERE Date(allDevData.{tables_with_timestamps[table_name]}) = '{one_day.date()}'"
+                    timestamp_ProdData = f" WHERE Date(allProdData.{tables_with_timestamps[table_name]}) = '{one_day.date()}'"
+                    row_result = extra_rows
+                if time_count == 2:
+                    one_month = today - timedelta(days=31)
+                    timestamp_DevData = f" WHERE Date(allDevData.{tables_with_timestamps[table_name]}) = '{one_month.date()}'"
+                    timestamp_ProdData = f" WHERE Date(allProdData.{tables_with_timestamps[table_name]}) = '{one_month.date()}'"
+                    if extra_rows["in_Prod_not_Dev"].values[0] is not None and extra_rows["in_Dev_not_Prod"].values[0] is not None:
+                        not_matched = True
+                        not_matched_info += " yesterday :"
+                if time_count == 3:
+                    six_month = today - timedelta(days=186)
+                    timestamp_DevData = f" WHERE Date(allDevData.{tables_with_timestamps[table_name]}) = '{six_month.date()}'"
+                    timestamp_ProdData = f" WHERE Date(allProdData.{tables_with_timestamps[table_name]}) = '{six_month.date()}'"
+                    if extra_rows["in_Prod_not_Dev"].values[0] is not None and extra_rows["in_Dev_not_Prod"].values[0] is not None:
+                        not_matched = True
+                        not_matched_info += " One Month :"
+                time_count += 1
+            else:
+                timestamp_DevData = ""
+                timestamp_ProdData = ""
+                time_iteration = False
+                has_timestamp = False
+
+            # Initializing query
+            extra_rows_query = f"""WITH
+            DevData AS (
+                SELECT
```

Comment:
so how much data does this query scan (check in BigQuery) for the cromwell_metadata table?

Reply:
412.7 MB with timestamps

Reply:
cool, that seems manageable, can you try not removing cromwell_metadata from the list of tables and see if you can run the check on it?